doc/source/contributing_code.rst

```diff
@@ -239,12 +239,12 @@ First we need to install dependencies. You'll need a mix of conda and pip instal
           'parmed' \
           'seaborn' \
           'scikit-learn' \
-          'tidynamics>=1.0.0'
+          'tidynamics>=1.0.0' \
+          'mda-xdrlib'
 
         # documentation dependencies
-        conda install -c conda-forge sphinx pybtex pybtex-docutils \
-            sphinxcontrib-bibtex mdanalysis-sphinx-theme sphinx-sitemap
-        python -m pip install
+        conda install -c conda-forge 'mdanalysis-sphinx-theme>=1.3.0'
+        python -m pip install docutils sphinx-sitemap sphinxcontrib-bibtex pybtex pybtex-docutils
```

Comment:
This is more closely replicating what CI does - I'm not sure if they are all conda installable, in which case we should just do that for everything?

Reply:
Yes, I think we can do everything with conda. Or at least I can do anything with micromamba